lib/bindings/python/rust/engine.rs

```diff
@@ -121,10 +123,16 @@ impl PythonServerStreamingEngine {
         generator: Arc<PyObject>,
         event_loop: Arc<PyObject>,
     ) -> Self {
+        let has_pycontext = Python::with_gil(|py| {
+            let callable = generator.bind(py);
+            callable_accepts_kwarg(py, callable, "context").unwrap_or(false)
```

Comment:
Just for my own understanding - this context will only be passed if the python request handler has an argument named exactly "context" and things like "ctx", etc. won't work as expected?

Reply:
Yes, the variable has to be named "context" on the generator method argument.

The interface after this PR is
* the current "request" argument is required and it is the first positional argument.
* the new "context" argument is optional and the binding only calls it as a keyword argument.

The interface is more strict for only two arguments. If we are sure we will never add a third argument, we can convert the "context" to the second positional argument in the future.